birdseye/server.py

```diff
@@ -13,6 +12,11 @@
 from birdseye.db import Call, Function, Session
 from birdseye.utils import all_file_paths, short_path, IPYTHON_FILE_PATH
 
+import argparse
+parser = argparse.ArgumentParser(description='Bird\'s Eye: A Python debugger with UI')
```

Comment:
Please use double quoted strings in cases like this to avoid having to escape single quotes

Reply:
Alright, I'll fix them.